assets/apps/dashboard/src/Components/Promotions.js

```diff
@@ -0,0 +1,26 @@
+/* global neveDash */
+import Promotion from './Promotion';
+const Promotions = () => {
+	if (!neveDash.promotions) {
+		return null;
+	}
+	if (1 > neveDash.promotions.length) {
+		return null;
+	}
+
+	return (
+		<div className="promotions">
+			{Object.keys(neveDash.promotions).map((slug, index) => {
+				return (
+					<Promotion
+						key={index}
```

Comment:
It's generally a bad practice to use the index of an array item as a key. We can use the `slug` here.